src/store/reducers/overview/overview.ts

```diff
@@ -6,14 +10,51 @@ export const overviewApi = api.injectEndpoints({
             queryFn: async ({paths, database}: {paths: string[]; database: string}, {signal}) => {
                 try {
                     const [data, ...additionalData] = await Promise.all(
-                        paths.map((p) => window.api.getDescribe({path: p, database}, {signal})),
+                        paths.map((p) =>
+                            window.api.getDescribe(
+                                {
+                                    path: p,
+                                    database,
+                                },
+                                {signal, concurrentId: 'overview'},
+                            ),
+                        ),
                     );
                     return {data: {data, additionalData}};
                 } catch (error) {
                     return {error};
                 }
             },
+            keepUnusedDataFor: 0,
```

Comment:
Overview may have a big size. So if user traverse through schema objects quickly enough, JS heap becomes huge and interface starts glitching. To prevent this I allow redux to get rid of previous data and not to keep it in cache.